R/ts_plot.R

```diff
@@ -1,8 +1,8 @@
 #' Time series decomposition and plot
 #'
-#' Convert csv file into a time series object, decompose it into trend, seasonality/cyclicity
+#' Convert a dataframe into a time series object, decompose it into trend, seasonality/cyclicity
 #' and noise/remainder/error and plot the raw data and the decomposed components.
-#' If the time series can't be decomposed, the function will just return the line chart of the raw data. 
+#' If the time series can't be decomposed, the function will just return the line chart of the raw data.
 #'
 #' @param data str: the path of the csv file
```

Comment:
```suggestion
#' @param data dataframe: should be data.frame or tibble
```
docstring is inconsistent with function input.